schema/crdb/README.adoc

```diff
@@ -7,28 +7,31 @@ This directory describes the schema(s) used by CockroachDB.
 
 We use the following conventions:
 
-* `schema/crdb/VERSION/up.sql`: The necessary idempotent migrations to
-  transition from the previous version of CockroachDB to this version. These
-  migrations will always be placed within one transaction per file.
-** If more than one change is needed per version, any number of files starting
-   with `up` and ending with `.sql` may be used. These files will be sorted in
-   lexicographic order before being executed, and each will be executed in a
+* `schema/crdb/VERSION/up*.sql`: Files containing the necessary idempotent
+  statements to transition from the previous version of the database schema to
+  this version.  All of the statements in a given file will be executed
+  together in one transaction; however, usually only one statement should
+  appear in each file.  More on this below.
+** If there's only one statement required, we put it into `up.sql`.
+** If more than one change is needed, any number of files starting with `up`
+   and ending with `.sql` may be used. These files will be sorted in
+   lexicographic order before being executed.  Each will be executed in a
    separate transaction.
 ** CockroachDB documentation recommends the following: "Execute schema
-   changes... in a single explicit transaction consisting of the single schema
```

Comment:
I'm not sure if the CockroachDB docs changed, but the text as-is here does not (quite) appear in the linked docs.  I only noticed because I was trying to learn more and looked for the source and had a hard time finding it.  There's an extra word "single" here.